index.html

```diff
@@ -6,37 +6,31 @@
 		<title>Thrilling Frontend Technical Challenge</title>
 
 		<link rel="icon" type="image/x-icon" href="/tech-challenge/favicon.ico" />
+		<link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+		<script src="https://cdn.jsdelivr.net/npm/lodash@4.17.15/lodash.min.js"></script>
+		<script src="search.js"></script>
+
 
 		<style>
 			body {
 				font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Helvetica Neue", Arial, sans-serif;
 			}
+			form {
+				padding: 20px;
+			}
 		</style>
 	</head>
 
 	<body>
-		<h1>Thrilling Frontend Technical Challenge</h1>
-
-		<h2>Open your Developer Tools Console to view output.</h2>
-
-		<script>
-			var request = new XMLHttpRequest();
-
-			request.open('GET', 'https://api.nytimes.com/svc/topstories/v2/science.json?api-key=Gwxln5M3geWlhR6UE0TY1FUWKSG3wCil');
-
-			request.onload = function() {
-				if (this.status >= 200 && this.status < 400) {
-					console.log(JSON.parse(this.response));
-				} else {
-					console.error(this);
-				}
-			};
+		<form id="searchForm">
 
-			request.onerror = function() {
-				console.error(this);
-			};
+		  <div class="form-group">
+		    <label for="searchBox">Enter Your Search Term</label>
+		    <input type="text" class="form-control" id="searchBox" placeholder="Search">
+		  </div>
+		  <button type="button" class="btn btn-primary" id="searchButton">Submit</button>
```

Comment:
consider changing input type to submit
```suggestion
		  <button type="submit" class="btn btn-primary">Submit</button>
```
See comment on line 60-62 on `search.js` below for further explanation.